### ISSUE TITLE
itvplayer plugin unable to find a stream

### ISSUE THREAD

**Issue by skiant** [chrippa/livestreamer#1068]
`livestreamer https://www.itv.com/itvplayer/itv best` returns
```
[cli][info] Found matching plugin itvplayer for URL https://www.itv.com/itvplayer/itv
error: No streams found on this URL: https://www.itv.com/itvplayer/itv
```

Windows 8.1
livestreamer.exe 1.12.2

I'm willing to dig and find more information to help out fixing this, but I don't know where to start.

**Comment by DanGoodman92** [comment]
Did you figure any of this out?

**Comment by skiant** [comment]
No, sorry. A quick look made me think the request made in https://github.com/chrippa/livestreamer/blob/develop/src/livestreamer/plugins/itvplayer.py#L61 is no longer returning what was expected, I'll have a more in-depth look later on.

**Comment by DanGoodman92** [comment]
Excellent, thank you for getting back to me. I look forward to seeing if
you come up with anything.

Kind Regards,
*Dan Goodman*

**Comment by peakie47** [comment]
I have the same issue on Linux (Ubuntu 14.04) using livestreamer 1.12.2.

**Comment by rb07** [comment]
**Skiant** is right, the request returns a faultcode of "`InvalidVodcrid`", no idea what that means (there is a Vodcrid in lines 174 - 178, but its not used in the request), but the plugin expects no faultcode, so it stops there returning `None`for streams.

I'm using the **itvhub** plugin shown here: [https://github.com/chrippa/livestreamer/issues/1233]()  This variation is working for live streams, but not for regular videos.
